packages/generator-adp/src/utils/parse-json-arg.ts

```diff
@@ -0,0 +1,41 @@
+import { isRecordOfStrings, isString } from './type-guards';
+
+/**
+ * Returns the first argument from a list of CLI arguments. If the first argument
+ * is not a string returns empty string.
+ *
+ * @param args The list of CLI command arguments.
+ * @returns The first parameter in the argument's list as string.
+ */
+export function getFirstArgAsString(args: string | string[]): string {
```

Comment:
All of the JSDocs should look the same or be similar in terms of structure.

Although this is optional and the benefits are not so much, I try to put the types for the properties in the JSDoc (see args).
Also, after the `@param {type} property` usually comes a dash ("-").

All of the above make the comments more sophisticated, helping both humans and machines understand your code.

```suggestion
/**
 * Returns the first argument from a list of CLI arguments. If the first argument
 * is not a string returns empty string.
 *
 * @param {string | string[]} args - The list of CLI command arguments.
 * @returns {string} The first parameter in the argument's list as string.
 */
export function getFirstArgAsString(args: string | string[]): string {
```

Reply:
fixed; align with jsdoc rules